www/js/insert-slide.js

```diff
@@ -153,24 +140,30 @@ modal.addFooterBtn('Close', 'tingle-btn tingle-btn--pull-right tingle-btn--dange
   modal.close();
 });
 
-// changes the tab to Dhan Guru Slides tab
-modal.addFooterBtn('Slides', 'tingle-btn tingle-btn-pull-left tingle-btn--default', () => {
-  // sets up the modal content (buttons, etc)
-  modal.setContent(slidePage);
-  // adds functionality to the buttons
-  isAnnouncementTab = false;
-  buttonOnClick();
-});
-
 // change tab to announcements
-modal.addFooterBtn('Announcement', 'tingle-btn tingle-btn-pull-left tingle-btn--default', () => {
-  // sets up modal content (box, etc)
-  modal.setContent(announcementPage);
-  isAnnouncementTab = true;
-  setLangSliderVal();
-  boxInputFunctionality();
+// changes to back when the tab is open
+// onclick is blank because it will change depending on what is being displayed
+modal.addFooterBtn(
+  'Announcement',
+  'tingle-btn tingle-btn-pull-left tingle-btn--default modal-tab-btn',
+  () => {},
+);
+const modalTabBtn = document.querySelector('.modal-tab-btn');
+modalTabBtn.addEventListener('click', () => {
+  if (!isAnnouncementTab) {
+    isAnnouncementTab = true;
+    // sets up modal content (box, etc)
+    modal.setContent(announcementPage);
```

Comment:
Can you make the 2 contents the same height? that way the element doesn't bounce.

Reply:
There would be a ton of whitespace/unused space

Reply:
make the announcement box bigger.